genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/ClaimsChecker.tsx

```diff
@@ -75,18 +81,39 @@ interface ClaimsCheckerProps {
         return <InlineAlert variant="notice" width="100%"><Heading>{totalIssues} issues need attention on Email {experienceNumber + 1}</Heading></InlineAlert>
     }
   }
-  
-    return (
-      <View>
-        {/* Status Alert */}
-          <Flex gap="size-100"  justifyContent="space-between">
-            {message(totalIssues)}
-          </Flex>
-  
-        {renderSection('Pre header', claims.pre_header)}
-        {renderSection('Header', claims.header)}
-        {renderSection('Body', claims.body)}
-      </View>
-    );
-  };
+
+  // Group violations by pod and field
+  const violationsByPodAndField: Record<string, Record<string, Violation[]>> = {}
+  for (const rawField of Object.keys(claims)) {
+    const field = rawField.replace(/pod\d+_/, '')
+    const match = rawField.match(/^pod(\d+)_/i);
+    const pod = match ? match[1] : '0';
```

Comment:
just want to doublecheck what this looks like in the UI for single pod requests. it shouldn't be called Pod0_header for ex, just "header"

Reply:
Hum. I didn't realize that, but assuming the logic works for both.
Let me try one and put a screen record here as well for single pod

Reply:
https://github.com/user-attachments/assets/3005745f-1d9d-42b6-ab17-12d969b5bf3e